frontend/src/components/editor/output/JsonOutput.tsx

```diff
@@ -75,6 +75,22 @@ function inferBestFormat(data: unknown): "tree" | "raw" {
   return typeof data === "object" && data !== null ? "tree" : "raw";
 }
 
+const CollapsibleTextOutput = (props: { text: string }) => {
+  const [isCollapsed, setIsCollapsed] = useState(true);
+  return (
+    <span className="cursor-pointer">
+      {isCollapsed ? (
+        <span onClick={() => setIsCollapsed(false)}>
+          {props.text.slice(0, 500)}
+          {props.text.length > 500 && "..."}
```

Comment:
this changes the default case such that all text is hidden at 500. that is ok w/ me, but just point that out. 

so if you have a lot of keys you want to expand, you'd have to click all of them (maybe not a common case)

Reply:
okay I'll add a comment there. this is making me think that having a dedicated JSON viewer / editor plugin would be nice to have...

Reply:
Yep that's what `mo.tree` is for.